code/script.js

```diff
@@ -1,13 +1,85 @@
 // All the DOM selectors stored as short variables
-const chat = document.getElementById('chat')
+const chat = document.getElementById("chat")
+const nameForm = document.getElementById("name-form")
+const inputwrapper = document.getElementById("input-wrapper")
 
 // Global variables, if you need any, declared here
 
 // Functions declared here
 
+const nameInput = document.getElementById("name-input")
+
+const handleNameInput = (event) => {
+  event.preventDefault()
+
+  showMessage(nameInput.value, "user")
+
+  nameInput.value = ""
+
+  setTimeout(()=> question1(), 1000)
+
+  
+
+  }
+
+  const question1 = () => {
+    showMessage("Hur ofta vill du spara?", "bot")
+    inputwrapper.innerHTML = `
+    <button id="button1">Lite varje månad</button>
+    <button id="button2">Ett engångsbelopp</button>`
+    document.getElementById("button1")
+    .addEventListener("click", () => question2())
+    document.getElementById("button2")
+    .addEventListener("click", () => answer1())
+
+  }
+
+  const question2 = () => {
+    showMessage("Strålande! Att spara lite varje månad gör att du sprider riskerna. Hur mycket vill du spara i månaden?", "bot")
+    inputwrapper.innerHTML = `
+    <button id="button3">Över 500 kr</button>
+    <button id="button4">Under 500 kr</button>`
+    document.getElementById("button3")
+    .addEventListener("click", () => question3())
+    document.getElementById("button4")
+    .addEventListener("click", () => answer1())
+  }
+
+  const question3 = () => {
+    showMessage("Så den sista och kanske viktigaste frågan! Vad brinner du mest för?", "bot")
+    inputwrapper.innerHTML = `
+    <button id="button5">Miljö & klimat</button>
+    <button id="button6">Sociala frågor</button>
+    <button id="button7">Jämställdhet</button>`
+    document.getElementById("button5")
+    .addEventListener("click", () => answer2())
+    document.getElementById("button6")
+    .addEventListener("click", () => answer2())
+    document.getElementById("button7")
+    .addEventListener("click", () => answer3())
+  }
+
+  const answer1 = () => {
+    showMessage("Hållbara aktier kanske passar dig!", "bot")
+    inputwrapper.innerHTML = `
+    <button onclick="window.location.href=('https://www.aktieinvest.se/hallbart-sparande/hallbara-aktier')">Läs mer om hållbara aktier!</button>`
+  }
+
+  const answer2 = () => {
+    showMessage("Hållbarhetsboxen kanske passar dig!", "bot")
+    inputwrapper.innerHTML = `
+    <button onclick="window.location.href=('https://www.aktieinvest.se/hallbarhetsboxen')">Läs mer om hållbarhetsboxen!</button>`
+  }
+
+  const answer3 = () => {
+    showMessage("Framtidsfeministens sparbox kanske passar dig!", "bot")
+    inputwrapper.innerHTML = `
+    <button onclick="window.location.href=('https://www.aktieinvest.se/framtidsfeministen')">Läs mer om Framtidsfeministens sparbox!</button>`
+  }
```

Comment:
If you were to use conditional statements and sent along a parameter, you could squish these three functions into one. It would look something like:

```
const giveFinalAdvice = usersChoice => {
  if (usersChoice === "environment") {
    //Show environment message
  } else if (usersChoice === "social") {
    //Show social message
  } else {
    //Show feminist alternative
  }
}
```